pages/01.metakube/01.release-notes/default.en.md

```diff
@@ -603,3 +603,7 @@ that assigns a default CPU and memory request to pods that do not define one exp
 #### 2018-08-01
 
 * MetaKube is now generally available
+
+#### 2018-09-30
```

Comment:
You probably want to use 2020 here ;)

Reply:
Thanks, updated

Reply:
Now the note should be somewhere at the top to make the timeline correct again :)

Reply:
Thanks again, updated